### PR TITLE
Update GOG.Galaxy.installer.yaml URL

### PR DESCRIPTION
**Pull request by rakleed**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/130477)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
The URL you provided does not point to the same version as the URL currently in the manifest.  The URL existing in the manifest should not be changed

```
PS D:\Git\winget-pkgs> Get-UrlPackageVersion 'https://webinstallers.gog-statics.com/download/GOG_Galaxy_2.0.exe'

Name                           Value
----                           -----
FileVersion                    2.0.0.2
ProductVersion                 2.0.0.2
Hash                           FD627D2C37BB5FA3540384FB2923DBFEFA1B222D8C34E14F9127C8971EFB093E
```

**Comment by rakleed** [comment]
@stephengillie the web installer is downloaded from the link I provided, which, after launch, automatically downloads the latest version of the full installer. I don't know if winget even supports this.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 12/15/2023 9:12:49 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   FD627D2C37BB5FA3540384FB2923DBFEFA1B222D8C34E14F9127C8971EFB093E
Expected hash: 612c3d962935b9b206b7619867355625f475abcf052f9dee7ac4521e235eaacf

File: manifests/g/GOG/Galaxy/2.0.73.27/GOG.Galaxy.installer.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ ProductCode: '{7258BA11-600C-430E-A759-27E2C691A335}_is1'
 ReleaseDate: 2023-11-02
 Installers:
 - Architecture: x86
-  InstallerUrl: https://gog-cdn-fastly.gog.com/open/galaxy/client/2.0.73.27/setup_galaxy_2.0.73.27.exe
+  InstallerUrl: https://webinstallers.gog-statics.com/download/GOG_Galaxy_2.0.exe
   InstallerSha256: 612C3D962935B9B206B7619867355625F475ABCF052F9DEE7AC4521E235EAACF
```
Comment (stephengillie) [inline manifests/g/GOG/Galaxy/2.0.73.27/GOG.Galaxy.installer.yaml:16]:
```suggestion
  InstallerSha256: FD627D2C37BB5FA3540384FB2923DBFEFA1B222D8C34E14F9127C8971EFB093E
```

**Comment by rakleed** [comment]
Why did you automatically add https://github.com/microsoft/winget-pkgs/issues?q=is%3Aopen+label%3ANeeds-Author-Feedback+sort%3Aupdated-desc label?

**Comment by rakleed** [comment]
@stephengillie so you still don’t support web installers?

**Comment by stephengillie** [comment]
@rakleed Not sure what you mean. We support many types of installers. 

This PR seems to have fallen between the cracks. Let me know if it should be closed again.

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

Hi @rakleed,

Is there an installer switch to complete the install automatically?

**Pull request closed without merging**